website/static/community.html

```diff
@@ -50,14 +50,14 @@ <h2>Issues</h2>
 
         <p>
             If you find an issue with Presto, please
-            <a href="https://github.com/facebook/presto/issues/new">report</a> it.
+            <a href="https://github.com/prestodb/presto/issues/new">report</a> it.
         </p>
 
         <h2>Contributing</h2>
 
         <p>
             You can also contribute fixes and new features as
-            <a href="https://github.com/facebook/presto/pulls">pull requests</a>.
+            <a href="https://github.com/prestodb/presto/pulls">pull requests</a>.
             You will need to sign a
             <a href="https://developers.facebook.com/opensource/cla">Contributor License Agreement</a>
```

Comment:
Not related to this PR, but we should also fix this and possible other stale/old links.

Reply:
Looks like I missed that one. I will push a commit shortly to add that to this batch of changes.

Reply:
Now this PR is merged, please create a separate PR. Thanks!